### PR TITLE
ELEMENTS-674: reduce memory footprint

### PR DESCRIPTION
**Pull request by afjusto**

### CODE REVIEW

**Comment by Gabez0r** [comment]
👍

File: nuxeo-document-preview.html [hunk 1]
```diff
@@ -232,10 +232,11 @@
               // Insert our previewer
               delete previewer.__templatizeOwner;
               this.templatize(previewer);
-              const instance = this.stamp();
+              let instance = this.stamp();
```
Comment (nelsonsilva) [inline nuxeo-document-preview.html]: this is a local variable so I'm guessing it won't really hold the reference to the instance...

Comment (afjusto) [reply nuxeo-document-preview.html]: yes, doesn't matter and was pushed by mistake 👍 ✅

**Comment by nelsonsilva** [comment]
👍

**Pull request merged**